src/main/java/com/google/api/codegen/config/DiscoGapicMethodConfig.java

```diff
@@ -162,8 +162,13 @@ static DiscoGapicMethodConfig createDiscoGapicMethodConfig(
     }
 
     Iterable<FieldConfig> requiredFieldConfigs =
-        DiscoGapicMethodConfig.createFieldNameConfigs(
-            DiscoGapicMethodConfig.getRequiredFields(
+        createFieldNameConfigs(
```

Comment:
You'll also want to do this below, for the `optionalFieldConfigs`. See `GapicMethodConfig.java:180`.

Reply:
Done.